garnetred/overlook-hotel
# Feature/loginvalidation

#### What's this PR do?
I successfully implemented Fetch in order to retrieve data from the server. I also created the logic for the login functionality, which I will further implement tomorrow. I finished building out all CSS and HTML pages for now. 
#### Where should the reviewer start (files changed)?
```
	modified:   src/domUpdates.js
	modified:   src/index.html
	modified:   src/index.js
	modified:   src/domUpdates.js
	modified:   src/css/_mixins.scss
	modified:   src/css/_variables.scss
	modified:   src/css/base.scss
```
#### How should this be manually tested?
This can be tested by running the page in the browser and removing the 'hide' class from the pages that you would like to see. There are a few nested pages within the customer and manager page sections as well, which would need to be unhidden. I'm debating if I want to change this formatting, which I include so as not to have to constantly rewrite the header depending on the user type.
#### Any background context you want to provide?

#### What are the relevant tickets?

#### Screenshots (if appropriate)

#### Questions:
I ended up moving all of my global variables to the domUpdates file, as well as the initial fetch request. I can't help but wonder if maybe I did the wrong thing in doing so... I'm also unsure if I have too many global variables (though they're just being declared and not initialized globally) and should work harder to decrease the number.
